cb-sol-cli/cmd/setup-tokens-config.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
+  CBG_URL: "",
```

Comment:
It could, but since it requires a lot of values I thought it would be more "user friendly" this way.
I don't even think this PR should be merged to _master_..